exonum/src/blockchain/builder.rs

```diff
@@ -71,7 +71,7 @@ impl BlockchainBuilder {
         let has_genesis_block = !Schema::new(&snapshot).block_hashes_by_height().is_empty();
 
         if has_genesis_block {
-            blockchain.dispatcher.restore_state(&snapshot)?;
+            blockchain.dispatcher.restore_state(&snapshot);
         } else {
             blockchain.create_genesis_block(self.genesis_config)?;
```

Comment:
Should this method still return a `Result` though? I don't think there are many (if any) scenarios, in which the error here can be meaningfully recovered from.